docs/uml/puml/.PlantUMLSequence.config

```diff
@@ -0,0 +1,26 @@
+hide circle
+hide footbox
+skinparam {
+    DefaultTextAlignment left
+    SequenceMessageAlignment center
+    wrapWidth 300
+    maxMessageSize 180
+}
+
+'Makes refs opaque like in the course textbook
+<style>
+Reference {
+  BackgroundColor white
+  HorizontalAlignment center
+}
+</style>
+
+'Centers text in participants so that things like <<class>> are centralised
+<style>
+Participant {
+  HorizontalAlignment center
+}
+</style>
+
+'Makes it so that the class does not have to be activated everytime for the activation bar to show up.
+autoactivate on
```

Comment:
Thanks for creating this `.config`  file! It allows us to create course-compliant diagrams and will be helpful for the rest of the group.